source/index.rst

```diff
@@ -180,6 +180,7 @@ Any file uploaded to ``play`` should be considered public and non-protected.
 
       Software Development Kits (SDK) <https://min.io/docs/minio/linux/developers/minio-drivers.html?ref=docs>
       Security Token Service (STS) <https://min.io/docs/minio/linux/developers/security-token-service.html?ref=docs>
+      Object Lambda <https://min.io/docs/minio/linux/developers/transform-data-with-object-lambda.html?ref=docs>
```

Comment:
The title here appears to be replaced with the page title from the file, I guess this one should match?